packages/zowe-explorer/src/utils/AuthUtils.ts

```diff
@@ -195,13 +191,9 @@ export class AuthUtils {
      * @returns {Promise<boolean>} a boolean representing whether token based auth is being used or not
      */
     public static async isUsingTokenAuth(profileName: string): Promise<boolean> {
```

Comment:
Curious if this (and the AuthHandler.isUsingTokenAuth) should be moved to the ProfileInfo APIs. What about the Config APIs? 😋